komodo-relational/src/main/java/org/komodo/relational/teiid/CachedTeiid.java

```diff
@@ -72,10 +89,10 @@
          */
         @Override
         public CachedTeiid create( final UnitOfWork transaction,
-                             final Repository repository,
-                             final KomodoObject parent,
-                             final String id,
-                             final RelationalProperties properties ) throws KException {
+                                   final Repository repository,
+                                   final KomodoObject parent,
+                                   final String id,
+                                   final RelationalProperties properties ) throws KException {
             return new CachedTeiidImpl(transaction, repository, id);
```

Comment:
Not going to answer each comment, but in general no the formatting changes are not absolutely necessary.  Maybe we should have someone periodically reformat poorly formatted files or add missing java docs, etc.  In general I see a lot of missing java docs, code warnings, etc.  Developers don't seem to be using a consistent set of eclipse preferences.